index.html

```diff
@@ -1,67 +1,214 @@
 <!DOCTYPE html>
-<html>
-  <head>
-     <link ref="stylesheet" href="./style.css" type="text/css">
-     <style>
-     body
-     {
-     background-color: blue;
-      }
+<html lang="en">
 
-     .navbar
-     {
-       display: inline-block;
-       font-size:60px;
-       padding: 10px;
-     }
-     .icon
-     {
-       height: 8%;
-       width: 8%;
-     }
-     .grid-container
-     {
-       display:grid;
-       grid-template-columns: 1fr 1fr;
-       grid-gap: 20px;
-     }
-     .thumbnail
-     {
-       display:block;
-       margin:2%;
-     }
-    </style>
+<head>
+  <title>Custom Video Player - Youtube Replica</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles/style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+</head>
 
-  </head>
-
-  <body>
-
-    <nav class="navbar">
-          <img class="icon" src=icon.png>
-          <a>Home</a>
-          <a>Trending</a>
-          <a>Help</a>
-          <a >About</a>
-    </nav>
-
-    <section class="grid-container">
-
-      <section class="mainvideo">
-        <iframe width="560" height="315"
-        src="https://www.youtube.com/embed/lJIrF4YjHfQ"
-        frameborder="0"
-        allow="picture-in-picture"
-        allowfullscreen></iframe>
+<body>
+  <header>
+  <nav class="navbar flex">
+    <div class="navElement flex">
+      <i id="hamburger" class="material-icons">menu</i>
+      <img class="logo" src="img/youtube-icon.png" alt="Youtube Logo">
+    </div>
+    <div class="navElement flex">
+      <div class="wrap">
+        <div class="search flex">
+          <input type="text" class="searchTerm" placeholder="Search">
+          <button type="submit" class="searchButton">
+            <i class="fa fa-search"></i>
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="navElement flex">
+      <div class="notification flex">
+        <i id="upload-icon" class="fas fa-upload"></i>
+      </div>
+      <div class="notification flex">
+        <i id="apps-icon" class="material-icons">apps</i>
+      </div>
+      <div class="notification flex">
+        <i id="notification-icon" class="fas fa-bell"></i>
+      </div>
+      <div class="notification flex">
+        <i id="user-icon" class="far fa-user-circle"></i>
+      </div>
+    </div>
+  </nav>
+</header>
+<main>
+  <div class="grid-container">
+    <div class="mainvideo">
```

Comment:
Use semantic HTML e.g. `<video>` instead of `<div>`

Reply:
We tried using the `<video>` tag but because Youtube uses an `<iframe>`tag and loads the entire frame for video player from their website, we can't use the `<video>` tag to embed a Youtube video. 

So instead of a `<div>`, we used the `<figure>` tag